src/stacktrace.cc

```diff
@@ -95,23 +95,25 @@ static void print_backtrace_symbol(int count, void *addr)
 	char *symbol;
 	int offset;
 	int status;
-
-	// dladdr() translates address to symbolic info.
-	dladdr(addr, &dlip);
+	int dl_ret;
+	int len = SYMBOL_MAXLEN;
 
 #if __SIZEOF_LONG__ == 4
 	pr_out("%2d [%#10lx] ", count, (unsigned long)addr);
 #else
 	pr_out("%2d [%#14lx] ", count, (unsigned long)addr);
 #endif
+	// dladdr() translates address to symbolic info.
+	dl_ret = dladdr(addr, &dlip);
+	if (dl_ret == 0) {
+		pr_out("unknown\n");
+		return;
+	}
```

Comment:
I think printing `?` is better than `unknown` so how about this?
```suggestion
        if (!dladdr(addr, &dlip)) {
                pr_out("?\n");
                return;
        }
```

Reply:
`?` seems better, I'll change it as you suggested